### PR TITLE
Updates CI with latest plugin_template changes.

### PR DESCRIPTION
**Pull request by dkliban**
Runs functional tests in container running Pulp.
Removes ruby bindings test job.

[noissue]

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
It also needs to remove the bindings job test files similar to what my (now closed) PR does https://github.com/pulp/pulpcore/pull/2852/files

File: .github/workflows/scripts/script.sh [hunk 1]
```diff
@@ -125,17 +108,17 @@ if [ -f $FUNC_TEST_SCRIPT ]; then
...
     
     else
-        pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulpcore.tests.functional -m parallel -n 8
-        pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional -m "not parallel"
-        pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulp_file.tests.functional -m parallel -n 8
-        pytest -v -r sx --color=yes --pyargs pulp_file.tests.functional -m "not parallel"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulpcore.tests.functional -m parallel -n 8"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional -m 'not parallel'"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulp_file.tests.functional -m parallel -n 8 --nightly"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --pyargs pulp_file.tests.functional -m 'not parallel' --nightly"
```
[the remark this answers is not in the document]
Comment (bmbouter) [reply .github/workflows/scripts/script.sh:121]: Ohhh good catch

[the remark this answers is not in the document]
Comment (dkliban) [reply .github/workflows/scripts/script.sh:121]: good eye. i'll update plugin_template and this PR.

File: .github/workflows/scripts/script.sh [hunk 2]
```diff
@@ -125,17 +108,17 @@ if [ -f $FUNC_TEST_SCRIPT ]; then
...
 
     
     else
-        pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulpcore.tests.functional -m parallel -n 8
-        pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional -m "not parallel"
-        pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulp_file.tests.functional -m parallel -n 8
-        pytest -v -r sx --color=yes --pyargs pulp_file.tests.functional -m "not parallel"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulpcore.tests.functional -m parallel -n 8"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional -m 'not parallel'"
+        cmd_prefix bash -c "pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs pulp_file.tests.functional -m parallel -n 8 --nightly"
```
Comment (mdellweg) [inline .github/workflows/scripts/script.sh:120]: still on left?

**Review by mdellweg — APPROVED** [review]

**Pull request merged**